src/components/Media/test.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
-import { shallow, mount } from 'enzyme';
+import { mount } from 'enzyme';
 
 import Media from './';
 
 describe('Media', () => {
   describe('Media.Item', () => {
-    it('supports custom tags', () => {
+    xit('supports custom tags', () => {
```

Comment:
Asked @mdespuits what this did: `xit`  is just an `ignore` version of `it`, since we intend to make custom tags work later.